src/main/java/no/entur/uttu/security/UserContextService.java

```diff
@@ -0,0 +1,23 @@
+package no.entur.uttu.security;
+
+/**
+ *  This interface provides authorization features to the app
```

Comment:
The documentation could state that this abstraction can be used to extend/customize authorization features.